ImperatorToCK3/Data_Files/configurables/converter_faiths.txt

```diff
@@ -1224,6 +1224,59 @@ hellenism_religion = {
             doctrine = doctrine_clerical_marriage_allowed
             doctrine = doctrine_clerical_succession_temporal_fixed_appointment
 
+            localization = {
+                #High God Name
+                HighGodName = mithra_name
+                HighGodNamePossessive = mithra_possessive
+                HighGodNameSheHe = CHARACTER_SHEHE_HE
+                HighGodHerselfHimself = CHARACTER_HIMSELF
+                HighGodHerHis = CHARACTER_HERHIS_HIS
+            }
+        }	
+	mithra_religion= {
```

Comment:
Sorry for not mentioning it in the issue, but this faith was supposed to be in the Zoroastrian group.
Could you make a new block like this (only `faiths` entry needed, nothing else) and move the faith you added there?
```
zoroastrianism_religion = {
	faiths = { }
}	
```